packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatusWheel/ProgressWheel.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useCallback, useState } from 'react';
+import styled, { css } from 'styled-components';
+import { animations, Tooltip } from '@trezor/components';
+import { ProgressContent, Container as ProgressContentContainer } from './ProgressContent';
+import { lighten } from 'polished';
+import { useSelector } from '@suite-hooks/useSelector';
+import {
+    selectCurrentCoinjoinWheelStates,
+    selectSessionProgressByAccountKey,
+} from '@wallet-reducers/coinjoinReducer';
+import { useDispatch } from '@suite-hooks/useDispatch';
+import { useCoinjoinSessionBlockers } from '@suite/hooks/coinjoin/useCoinjoinSessionBlockers';
+import { goto } from '@suite-actions/routerActions';
+import { Translation } from '@suite-components/Translation';
+
+const ProgressIndicator = styled.div`
+    position: absolute;
+    top: 11px;
+    width: 94px;
+    height: 94px;
+    background: conic-gradient(#ffffff00 20deg, #cccccc);
+    border-radius: 50%;
+    font-size: 15px;
+    animation: ${animations.DELAYED_SPIN} 2.3s cubic-bezier(0.34, 0.45, 0.17, 0.87) infinite;
+`;
+
+const Wheel = styled.div<{
+    progress: number;
+    isPaused: boolean;
+    isHoverDisabled: boolean;
+    hasCriticalError: boolean;
+    isAccountEmpty: boolean;
+    isWithoutProgressOutline: boolean;
+    isStartable: boolean;
+}>`
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 94px;
+    height: 94px;
+    border-radius: 50%;
+    background: ${({ theme, progress }) =>
+        `conic-gradient(${theme.BG_GREEN} ${3.6 * progress}deg, ${theme.STROKE_GREY}a1 0)`};
```

Comment:
Concatenating the value from theme with `a1` string looks too fragile and cryptic to me, what about using rgba instead?
```suggestion
        `conic-gradient(${theme.BG_GREEN} ${3.6 * progress}deg, rgba(${theme.STROKE_GREY}, 0.5) 0)`};
```

Reply:
Ideally I would not even change opacity, at least we are not doing that often for color mixing now. Also `rgba()` does not use HEX :'( I can use the color picker and simply hardcode a heeded color I guess

Reply:
added `opacify()` instead 5455abfaf

Reply:
looks like you opacify the value on the other place but not here